tracer/build/_build/Build.Utilities.cs

```diff
@@ -492,6 +492,22 @@ private static MSBuildTargetPlatform GetDefaultTargetPlatform()
 
         return MSBuildTargetPlatform.x64;
     }
+    
+    private static string GetDefaultRuntimeIdentifier()
+    {
```

Comment:
Surprisingly, there's no utility which returns an acceptable `RuntimeIdentifier`, I copied this function from the PR this one is inspired, but it seems that the build has been hardcoded most of the time!

I indeed omitted `musl-x64` and `musl-arm64`, since this is only going to be used for R2R, but I can add them

Reply:
Added runtime identifiers for Alpine so this function can be reused, not gonna change the existing code tho, that'd be in another PR